makuai-corp-site/layouts/partials/footer.html

```diff
@@ -0,0 +1,27 @@
+<footer class="footer">
+    <div class="footer__nav">
+        <div class="footer__nav-column">
+            <a href="/" class="footer__nav-link">ホーム</a>
+            <a href="/curriculum" class="footer__nav-link">カリキュラム</a>
+            <a href="/introduction" class="footer__nav-link">学生の紹介</a>
+        </div>
+        <div class="footer__nav-column">
+            <a href="/company" class="footer__nav-link">スポンサー要領</a>
+            <a href="/studentrecruit" class="footer__nav-link">エンジニアをお探しの企業様</a>
+            <a href="/support" class="footer__nav-link">支援</a>
+        </div>
+        <div class="footer__nav-column">
+            <a href="/news" class="footer__nav-link">ニュース</a>
+            <a href="/contact" class="footer__nav-link">問い合わせ</a>
+        </div>
+        <div class="footer__nav-column">
+            <a href="/privacy" class="footer__nav-link">個人情報保護方針</a>
+            <a href="/privacy#statement" class="footer__nav-link">個人情報の取り扱いについて</a>
+            <a href="/privacy#sookies" class="footer__nav-link">クッキー等の利用について</a>
```

Comment:
```suggestion
            <a href="/privacy#cookies" class="footer__nav-link">クッキー等の利用について</a>
```
`/privacy`の方ではidがcookiesなのでここもそれに合わせた方がいいと思います。